tmt/steps/finish/__init__.py

```diff
@@ -104,28 +105,61 @@ def go(self) -> None:
             self.actions()
             return
 
-        # Go and execute each plugin on all guests
+        # Prepare guests
+        guest_copies: List[Guest] = []
+
         for guest in self.plan.provision.guests():
             # Create a guest copy and change its parent so that the
-            # operations inside finish plugins on the guest use the
-            # finish step config rather than provision step config.
+            # operations inside prepare plugins on the guest use the
```

Comment:
Is this correct? `prepare`? this is the code for finish?

Reply:
Fixed in 0c0f5892